content/post/AG2021.en.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "2021 general assembly review"
+date: 2021-09-23
+draft: false
+tags: [general-assembly]
+---
+
+Like every year, we hold our general assembly. On the 4th of September, we were gathered in the Nantes area, with a few remote members, to look back upon the past year and decide on the association stances for the year to come.
+
+### A year dampened by the Covid pandemic
+
+Same as everyone, we saw our outside activity slow down; we spoke at less conferences, but still had as many interviews with journalists.
+
+Less conferences and workshops doesn't mean **no** conferences or workshops. We had a few meaningful encounters:
+
+- at Lycée Blaise Pascal in Rouen, with the class of seconde 8 in SNT (Digital sciences and technology) lesson, and their teacher Yves Meret. This speech was briefly featured in the France 2 show Envoyé Spécial on 29/04/2021;
+- an interview with U+039b, one of the founders of the association, in the other France 2 show Cash Investigation;
+- participating in the France Inter podcast with Xavier de La Porte, "Le code a changé" on 07/12/2020, broadcasted on France Inter in summer 2021;
+- a lecture in the Foyer des Jeunes Travailleurs in Batignoles...
+
+### A year of consolidating the platform
+
+The εxodus platform runs very well and the time we didn't spend in conferences and workshop allowed us to step back and rest a bit.
+
+It also allowed us to consolidate the platform, on the development level with small interface improvements, as well as big improvements on the code and database side.
+
+We had to face clever clogs who wanted to play with our nerves, but these are made of steel and we were able to counter these attacks and implement preventive action.
+
+A few figures:
+
+- During the month of August, we crossed the threshold of 100.000 unique applications analysed by the platform;
+- In September, it's the threshold of 200.000 generated reports that we crossed;
+- The platform now knows more than 400 trackers signatures;
+
+- 42.
+
+### A welcoming year
+
+Even if we did little fieldwork this year, the field came to us and the broadcasting and rebroadcasting of the "Le code a changé" podcast changed our exposure.
+
+This way, we welcomed a few members:
+
+- BateauSurLeau
+- Jean-Baptiste
+- Kiwibleu
+- Philémon
+- Xakan
+
+### What now?
+
+As you probably know, MeTaL_PoU reached the end of the 3 years a person can spend being the association president (reread the statutes!). We heartily thank her for these 3 years which saw the association grow in these proportions we weren't expecting.
```

Comment:
```suggestion
As you probably know, MeTaL_PoU reached the end of the 3 years a person can spend being the association president (reread the statutes!). We heartily thank her for these 3 years which saw the association grow in proportions we weren't expecting.
```
Grows refer to me to the number of people here, but I don't think it grew this much regarding that, though?